src/main/java/courseManager/AssessmentInstance.java

```diff
@@ -0,0 +1,67 @@
+package courseManager;
+
+import java.time.LocalDate;
+
+public class AssessmentInstance {
+    private String assessmentInstanceName;
+    private LocalDate date;
+    private double mark;
+    private boolean isCommitted;
+    private boolean isSubmitted;
+
+    /**
+     * Create a new AssessmentInstance object with the given name and date
+     * @param assessmentInstanceName the name of the assessment instance
+     */
+
+    public AssessmentInstance(String assessmentInstanceName) {
+        this.assessmentInstanceName = assessmentInstanceName;
+        this.isCommitted = false;
+        this.isSubmitted = false;
```

Comment:
as of now, yes this is intended behavior. I didn't want to force the use case to already provide a date and mark for an Assessment Instance, so they would be initialized to 0. I now added this.mark = null and also changed the return type of the getters.
Let me know if you have a more elegant solution since null values can create some problems!